legate/core/context.py

```diff
@@ -29,16 +30,16 @@ class ResourceConfig(object):
         "max_shardings",
     ]
 
-    def __init__(self):
+    def __init__(self) -> None:
         self.max_tasks = 1_000_000
         self.max_mappers = 1
         self.max_reduction_ops = 0
         self.max_projections = 0
         self.max_shardings = 0
 
 
-class ResourceScope(object):
-    def __init__(self, context, base, category):
+class ResourceScope:
+    def __init__(self, context, base, category) -> None:
```

Comment:
Just curious, is `-> None` implicit if not specified? It seems to be added onto some `__init__` methods but not on others.

Reply:
It's not implicit, but this file itself is not yet checked by mypy. I did add a few types here provisionally to make checking  the other files more straightforward.